tutorial/practicals/LFRic/building_code/1_simple_kernels/part1/Makefile

```diff
@@ -70,42 +70,46 @@ EXAMPLE_OBJ = $(DRIVER_OBJ) $(ALGORITHM_OBJ) $(PSY_OBJ) $(KERNEL_OBJ)
 # Targets
 .DEFAULT_GOAL := compile
 
-.PHONY: compile transform clean
+.PHONY: transform compile run clean allclean
+
+# Test PSyclone target for continuous integration
+transform: $(PSY_SRC) $(ALGORITHM_SRC)
 
 compile: transform $(EXEC)
 
 run: compile
 	$(EXEC)
 
 $(EXEC): $(LFRIC_LIB) $(EXAMPLE_OBJ)
-	$(F90) $(F90FLAGS) $(EXAMPLE_OBJ) -o $(EXEC) -L$(LFRIC_PATH) -l$(LFRIC_NAME)
+	$(F90) $(F90FLAGS) $(LFRIC_INCLUDE_FLAGS) $(EXAMPLE_OBJ) \
+	-o $(EXEC) -L$(LFRIC_PATH) -l$(LFRIC_NAME)
 
 $(LFRIC_LIB):
 	$(MAKE) -C $(LFRIC_PATH)
 
 # Dependencies
-$(DRIVER_OBJ): $(ALGORITHM_OBJ) $(PSY_OBJ) $(KERNEL_OBJ)
-$(ALGORITHM_OBJ): $(PSY_OBJ) $(KERNEL_OBJ)
-$(PSY_OBJ): $(KERNEL_OBJ)
+$(DRIVER_OBJ): $(ALGORITHM_OBJ) $(PSY_OBJ) $(KERNEL_OBJ) $(LFRIC_LIB)
```

Comment:
You don't need LFRIC_LIB on the end of the first three lines here because the fourth one ensures that KERNL_OBJ depends on it.

Reply:
You're right, corrected :)